cafelatte/cafelatte.py

```diff
@@ -0,0 +1,24 @@
+from coffee_time.coffee import Coffee
+import math
+
+
+class Cafelatte(Coffee):
+    """
+    This is a derived class of Coffee
+    """
+
+    def __init__(self, coffee_spoons, sugar, water, milk):
+        super().__init__(coffee_spoons=coffee_spoons, sugar=sugar, water=water)
+        self.milk = milk
+
+    def prepare_coffee(self):
+        cantitate_totala_cafea = super().prepare_coffee() + self.milk
+        return cantitate_totala_cafea
+
+    def prepare_time_coffee(self):
+        time_to_prepare = math.pi*math.sin(self.coffee_spoons) + math.cos(self.sugar) + self.water + 2 * self.milk
```

Comment:
Method should have inherited prepare_time_coffee from Coffee class + milk

Reply:
But the requirement was to print at the end using just this method...